### PR TITLE
Implement accordion component

### PR DESCRIPTION
**Pull request by picklelo**
Test code:

```python
"""Welcome to Reflex! This file outlines the steps to create a basic app."""
from rxconfig import config

import reflex as rx
import reflex.components.radix.themes as rdxt
import reflex.components.radix.primitives as rdxp


docs_url = "https://reflex.dev/docs/getting-started/introduction"
filename = f"{config.app_name}/{config.app_name}.py"
import random

data = [
    {"name": "Page A", "uv": 4000, "pv": 2400, "amt": 2400},
    {"name": "Page B", "uv": 3000, "pv": 1398, "amt": 2210},
    {"name": "Page C", "uv": 2000, "pv": 9800, "amt": 2290},
    {"name": "Page D", "uv": 2780, "pv": 3908, "amt": 2000},
    {"name": "Page E", "uv": 1890, "pv": 4800, "amt": 2181},
    {"name": "Page F", "uv": 2390, "pv": 3800, "amt": 2500},
    {"name": "Page G", "uv": 3490, "pv": 4300, "amt": 2100},
]

data01 = [
    {"x": 100, "y": 200, "z": 200},
    {"x": 120, "y": 100, "z": 260},
    {"x": 170, "y": 300, "z": 400},
    {"x": 170, "y": 250, "z": 280},
    {"x": 150, "y": 400, "z": 500},
    {"x": 110, "y": 280, "z": 200},
]

data02 = [
    {"name": "Group A", "value": 400},
    {"name": "Group B", "value": 300},
    {"name": "Group C", "value": 300},
    {"name": "Group D", "value": 200},
    {"name": "Group E", "value": 278},
    {"name": "Group F", "value": 189},
]


class State(rx.State):
    """The app state."""

    count = 0

    def increment(self):
        """Increment the count."""
        self.count += 1

    def decrement(self):
        """Decrement the count."""
        self.count -= 1

    def random(self):
        """Randomize the count."""
        self.count = random.randint(0, 100)

def index() -> rx.Component:
    return rdxt.theme(
        rdxt.container(
            rdxt.flex(
                rdxp.accordion(
                    [
                        ("Is it accessible?", rdxt.button("Test button")),
                        ("Is it unstyled?", "Yes. It's unstyled by default, giving you freedom over the look and feel."),
                        ("Is it finished?", "It's still in beta, but it's ready to use in production.")
                    ],
                    type="single",
                    collapsible=True,
                    width="300px",
                ),
                rdxt.heading(State.count),
                rdxt.flex(
                    rdxt.button("Decrement", on_click=State.decrement),
                    rdxt.button(
                        "Randomize",
                        on_click=State.random,
                    ),
                    rdxt.button("Increment", on_click=State.increment),
                    direction="row",
                ),
                rx.recharts.area_chart(
                    rx.recharts.area(
                        data_key="uv",
                    ),
                    rx.recharts.x_axis(data_key="name"),
                    rx.recharts.y_axis(),
                    data=data,
                ),
                rx.recharts.scatter_chart(
                    rx.recharts.scatter(
                        data=data01,
                    ),
                    rx.recharts.x_axis(data_key="x", type_="number"),
                    rx.recharts.y_axis(data_key="y"),
                ),
                rx.recharts.pie_chart(
                    rx.recharts.pie(
                        data=data02,
                        data_key="value",
                        name_key="name",
                        cx="50%",
                        cy="50%",
                        label=True,
                        color_scheme="amber",
                    )
                ),
                # rx.data_editor(
                #     columns=columns,
                #     data=data,
                # ),
                direction="column",
                padding="1em",
                border_radius="1em",
                box_shadow="lg",
                gap="4",
            ),
            padding_y="5em",
            font_size="2em",
            text_align="center",
        )
    )


# Add state and page to the app.
app = rx.App(
    theme=rdxt.theme(
        appearance="dark", has_background=True, radius="none", accent_color="crimson"
    ),
)

app.add_page(index)
app.compile()

```

### CODE REVIEW

File: reflex/.templates/jinja/web/pages/base_page.js.jinja2 [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% import 'web/pages/utils.js.jinja2' as utils %}
+/** @jsxImportSource @emotion/react */
```
Comment (masenf) [inline reflex/.templates/jinja/web/pages/base_page.js.jinja2:2]: Not sure why, but in my app this is rendering as

```javascript
/** @jsxImportSource @emotion/react */import { Fragment } from "react"
```

It still seems to work, somehow

File: reflex/style.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def convert(style_dict):
     var_data = None  # Track import/hook data from any Vars in the style dict.
     out = {}
     for key, value in style_dict.items():
-        key = format.to_camel_case(key)
+        key = format.to_camel_case(key, allow_hyphens=True)
```
Comment (masenf) [inline reflex/style.py:51]: what is this intending to support? this breaks the code that i was working on to support things like `::-webkit-scrollbar` as pseudo elements

File: reflex/components/radix/primitives/accordion.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    """An accordion component."""
+
+    tag = "Trigger"
+
+    alias = "RadixAccordionTrigger"
+
+    style: Style = Style(
+        {
+            "fontFamily": "inherit",
+            "backgroundColor": "white",
```
Comment (masenf) [inline reflex/components/radix/primitives/accordion.py:136]: probably should use a theme color here

File: reflex/components/graphing/recharts/cartesian.py [hunk 4]
```diff
@@ -195,6 +196,10 @@ class Cartesian(Recharts):
     # The type of icon in legend. If set to 'none', no legend item will be rendered. 'line' | 'plainline' | 'square' | 'rect'| 'circle' | 'cross' | 'diamond' | 'star' | 'triangle' | 'wye' | 'none'optional
     # legend_type: Var[LiteralLegendType]
 
+    def _apply_theme(self, theme: Component):
+        self.stroke = Var.create(f"var(--accent-8)")
```
Comment (masenf) [inline reflex/components/graphing/recharts/cartesian.py:200]: This doesn't seem to be working... when i change the accent color, it does not affect the charts

File: reflex/style.py (hunk 2 above)
Comment (picklelo) [reply reflex/style.py:51]: I needed this for certain `aria-label` props, that were getting converted automatically to`ariaLabel` and messing it up. I'll see if I can remove this

**Review by Alek99 — APPROVED** [review]

**Pull request merged**